src/apps/material/material.dev.tsx

```diff
@@ -12,28 +12,43 @@ export default {
       control: { type: "text" }
     },
     materialHeaderAuthorByText: {
-      name: "Af forfatter",
+      name: "By (author)",
       defaultValue: "Af ",
       control: { type: "text" }
     },
     periodikumSelectYearText: {
-      name: "År",
+      name: "Year",
       defaultValue: "År",
       control: { type: "text" }
     },
     periodikumSelectWeekText: {
-      name: "Uge",
+      name: "Week",
       defaultValue: "Uge",
       control: { type: "text" }
     },
     reserveBookText: {
-      name: "Reserve book",
+      name: "Reserve",
       defaultValue: "RESERVER BOG",
       control: { type: "text" }
     },
-    fineOnBookshelfText: {
-      name: "Fine on bookshelf",
-      defaultValue: "FINE PÅ HYLDEN",
+    findOnBookshelfText: {
+      name: "Find on bookshelf",
+      defaultValue: "FIND PÅ HYLDEN",
+      control: { type: "text" }
+    },
+    descriptionHeadlineText: {
+      name: "Description headline",
+      defaultValue: "Beskrivelse",
+      control: { type: "text" }
+    },
+    baseUrlText: {
+      name: "Base search url",
+      defaultValue: "https://bibliotek.dk/search",
```

Comment:
Consider if the default value be the url of the search app in storyboook.

Reply:
That could actually be nice! I added it :)